sdk/identity/azure-identity/tests/test_managed_identity_client.py

```diff
@@ -36,15 +36,15 @@ def test_caching():
         request_factory=lambda _, __: HttpRequest("GET", "http://localhost"), transport=transport
     )
 
-    token = client.get_cached_token(scope)
+    token, refresh_on = client.get_cached_token(scope)
     assert not token
```

Comment:
We should probably add some `refresh_on` checking with `assert refresh_on is None` here.